src/psyclone/tests/domain/lfric/lfric_domain_kernels_test.py

```diff
@@ -289,16 +289,11 @@ def test_psy_gen_domain_kernel(dist_mem, tmpdir, fortran_writer):
     # we require a mesh object.
     assert "type(mesh_type), pointer :: mesh => null()" in gen_code
     assert "mesh => f1_proxy%vspace%get_mesh()" in gen_code
-    assert "integer(kind=i_def) ncell_2d_no_halos" in gen_code
+    assert "integer(kind=i_def) :: ncell_2d_no_halos" in gen_code
     assert "ncell_2d_no_halos = mesh%get_last_edge_cell()" in gen_code
 
     # Kernel call should include whole dofmap and not be within a loop
-    if dist_mem:
-        expected = "      ! call kernels and communication routines\n"
-    else:
-        expected = "      ! call our kernels\n"
-    assert (expected + "      !\n"
-            "      call testkern_domain_code(nlayers_f1, ncell_2d_no_halos, "
+    assert ("    call testkern_domain_code(nlayers_f1, ncell_2d_no_halos, "
```

Comment:
Please extend to check that it is not within a loop.

Reply:
Done